http.py

```diff
@@ -131,7 +133,8 @@ def send_request(self):
             self.sock.send(self.request)
         except socket.error as exc:
             print exc
-        self.get_response()
+        finally:
```

Comment:
thats how it was before since the exc was just being printed, it was being called anyways